### PR TITLE
[docs] link to splash screen reference

### PR DESCRIPTION
**Pull request by kaycebasques**

### CODE REVIEW

File: lighthouse-core/audits/splash-screen.js [hunk 1]
```diff
@@ -29,7 +29,9 @@ class SplashScreen extends MultiCheckAudit {
       name: 'splash-screen',
       description: 'Configured for a custom splash screen',
       failureDescription: 'Is not configured for a custom splash screen',
-      helpText: 'A default splash screen will be constructed for your app, but satisfying these requirements guarantee a high-quality [splash screen](https://developers.google.com/web/updates/2015/10/splashscreen) that transitions the user from tapping the home screen icon to your app\'s first paint',
+      helpText: 'Meet these requirements to ensure a high-quality experience when ' +
```
Comment (patrickhulce) [inline lighthouse-core/audits/splash-screen.js]: maybe its a good time to rephrase this a bit, "Meet these requirements" diverges quite a bit from our usual style here.

wdyt about something like "A themed splash screen ensures a high-quality experience when..."?

**Comment by sexywife** [comment]
False

**Comment by sexywife** [comment]
False

File: lighthouse-core/audits/splash-screen.js (hunk 1 above)
Comment (sexywife) [reply lighthouse-core/audits/splash-screen.js]: False

Comment (paulirish) [reply lighthouse-core/audits/splash-screen.js]: ping @kaycebasques

Comment (kaycebasques) [reply lighthouse-core/audits/splash-screen.js]: done

**Pull request merged**